### PR TITLE
Second part of fixes

### PR DESCRIPTION
**Pull request by theAsmodai**
Fixed some Cmd_ functions, fixed messages sending in ClientPutInServer forward, added unrolled (un)munge2 functions, added one overflow check, fixed fullupdate spam from not active clients.

### CODE REVIEW

File: rehlds/engine/sv_main.cpp [hunk 1]
```diff
@@ -493,6 +493,14 @@ void SV_AllocPacketEntities(client_frame_t *frame, int numents)
 {
 	if (frame)
 	{
+#ifdef REHLDS_FIXES
+		// only alloc for max possible numents
+		if (!frame->entities.entities)
+		{
```
Comment (dreamstalker) [inline rehlds/engine/sv_main.cpp]: Please use some constant of #define instead of magic numbers

File: rehlds/engine/sv_main.cpp [hunk 2]
```diff
@@ -493,6 +493,14 @@ void SV_AllocPacketEntities(client_frame_t *frame, int numents)
 {
 	if (frame)
 	{
+#ifdef REHLDS_FIXES
+		// only alloc for max possible numents
+		if (!frame->entities.entities)
+		{
+			frame->entities.num_entities = 256;
+			frame->entities.entities = (entity_state_t *)Mem_ZeroMalloc(sizeof(entity_state_t) * 256);
```
Comment (dreamstalker) [inline rehlds/engine/sv_main.cpp]: Please use some constant of #define instead of magic numbers

**Comment by dreamstalker** [comment]
Impressive! Thx for the PR!
Before I merge this I would ask you to squash all your commits into one, to avoid 'tons of commits' problem in the repo.

**Comment by theAsmodai** [comment]
All done.

**Pull request merged**